stestr/commands/load.py

```diff
@@ -180,7 +181,13 @@ def make_tests():
             case = testtools.DecorateTestCaseResult(case, decorate)
             yield (case, str(pos))
 
-    case = testtools.ConcurrentStreamTestSuite(make_tests)
+    if serial:
+        for _, stream in enumerate(streams):
```

Comment:
Is there a reason we need to use enumerate here instead of just doing `for stream in streams:` ?

Reply:
ah, yes. I should work. I just copy the code from `make_tests()` and modified it :-p